using-kubecost/navigating-the-kubecost-ui/savings/container-request-right-sizing-recommendations.md

```diff
@@ -16,6 +16,14 @@ The container request right-sizing recommendations page shows a tabular view of
 
 Recommendations from Kubecost can including both increasing, decreasing, as well as establishing net new resource requests. When Kubecost recommends a decrease in resource requests, this results in savings opportunities. The _EST. SAVINGS_ field (Estimated Savings) shows the estimated amount of monthly savings which can be achieved by adopting the recommendations. The amount of estimated savings depends on the Profile and other [configuration options](#configuring-request-right-sizing-recommendations) which are explained below. The container request right-sizing recommendations table is sorted by default in descending order according to the estimated savings available.
 
+The request sizing algorithm calculates monthly CPU and RAM savings by comparing current costs with projected costs for the recommended resource allocation. Here's how it works:
```

Comment:
This is just for savings estimation, can we make that clear?

Reply:
fixed